pkg/apis/extensions/install/install.go

```diff
@@ -105,3 +112,19 @@ func interfacesFor(version unversioned.GroupVersion) (*meta.VersionInterfaces, e
 		return nil, fmt.Errorf("unsupported storage version: %s (valid: %v)", version, g.GroupVersions)
 	}
 }
+
+func addVersionsToScheme(externalVersions ...unversioned.GroupVersion) {
+	// add the internal version to Scheme
+	extensions.Register()
```

Comment:
In the new terminology, it was "registered" already, right?
It is being "enabled" now?